tests/publiccloud/xfsprepare.pm

```diff
@@ -73,15 +99,26 @@ sub create_config {
     ## Profile config file containing device and mount point definitions
     assert_script_run("echo 'export TEST_DIR=$mnt_xfs' >> $CONFIG_FILE");
     assert_script_run("echo 'export SCRATCH_MNT=$mnt_scratch' >> $CONFIG_FILE");
-    assert_script_run("echo 'export TEST_DEV=${device}1' >> $CONFIG_FILE");
-    assert_script_run("echo 'export SCRATCH_DEV=${device}2' >> $CONFIG_FILE");
+    assert_script_run("echo 'export TEST_DEV=${part}1' >> $CONFIG_FILE");
+    assert_script_run("echo 'export SCRATCH_DEV=${part}2' >> $CONFIG_FILE");
     # Add optional mkfs options
     my $mkfs_options = get_var('XFS_MKFS_OPTIONS', '');
     $mkfs_options .= " -m reflink=1" if (get_var('XFS_TESTS_REFLINK', 0) == 1);
     $mkfs_options =~ s/^\s+|\s+$//g;    # trim string (left and right)
     assert_script_run("echo 'MKFS_OPTIONS=\"$mkfs_options\"' >> $CONFIG_FILE") unless ($mkfs_options eq '');
 }
 
+sub get_unused_nvme_device {
+    # Get all disks and partitions and return the first device, which has no parititons
```

Comment:
please move this line above, since it is description of whole function not single line which goes below . in current place it confusing and from the beginning I thought that it is just about next line ...

Reply:
done